app/controllers/admin/dashboard_controller.rb

```diff
@@ -1,6 +1,12 @@
 class Admin::DashboardController < Admin::BaseController
 
   def show
+    @orders = Order.all
+  end
 
+  def update
+    order = Order.find(params[:id])
+    order.ship_and_fulfill
+    redirect_to "/admin/dashboard"
```

Comment:
Ship and fulfill method is in order model: allows all item_orders to have their status switched to fulfilled and changes order status to shipped.